backend/docker-entrypoint.sh

```diff
@@ -3,6 +3,5 @@
 bash wait-for-it db:5432
 
 python3 manage.py migrate
-python3 manage.py create_admin hello@codeforpoznan.pl cfp123 --django-admin
-python3 manage.py create_admin ola@pah.org.pl pah123
+python3 manage.py populate_database
```

Comment:
I would prefer to have that as a separate command instead of part of starting backend container and would also prefer to leave `create_admin` as a separate script.

Reply:
If you think it will help - no problem. 
Can you please describe a scenario, where it would be required to run those two scripts separately?
Surely not on production  (password public). So this will be for development only.
And if it's only for development - than having two separate seed files makes little sense..

What do you think?

Reply:
We discuss that offline with @arturtamborski and we agreed, that we want to have a separate script to create super users and another one to create objects from factories. Also as you limited number of instances created using factories when running `populate_database` it would make no sense to run that script whenever you start backend container.

Reply:
Why do you want to have separate script for user creation?

How about actually dropping the entire schema on container start? This will make dev environment repeatable.

Reply:
Droping schema could be achived by `docker-compose rm db`. Personally believe that `make start` should not tear apart data available in previous runs.